lib/KCard/index.vue

```diff
@@ -13,12 +13,13 @@
       <slot name="title"></slot>
     </template>
     <template #default>
-      <div class="thumbnail">
+      <div :class="isThumbnailPresent ? 'thumbnail' : null">
         <!-- 
           Render KImg even if thumbnailSrc is not provided since in that case
           KImg takes care of showing the gray placeholder area.
         -->
         <KImg
+          v-if="thumbnailSrc"
```

Comment:
Relatedly to my other comments, let's remove this line (see the comment on line 17-20)